### PR TITLE
Update beersmith2 to 2.3.7

### PR DESCRIPTION
**Pull request by welly**

### CODE REVIEW

**Comment by welly** [comment]
Hmm.. the fail is strange. The URL of the download has changed, so _amazonaws.com/BeerSmith2-2_ is no longer valid for this. Not sure why this would be flagged to fail.

File: Casks/beersmith2.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 cask 'beersmith2' do
-  version '2.2.12'
-  sha256 'e4aa64af9e9c65e0cfe132de67ab580d0c3b2f527d598fcda0d9b2cc84b74da6'
+  version '2.3.7'
+  sha256 '116b1f9f767a031bb6daa5432fad54b0318fec27c0e316ca4ed2485997686011'
 
   # amazonaws.com/BeerSmith2-2 was verified as official when first introduced to the cask
-  url "https://s3.amazonaws.com/BeerSmith2-2/BeerSmith#{version.dots_to_underscores}_64.dmg"
+  url "http://s3.amazonaws.com/beersmith2-3/BeerSmith#{version.dots_to_underscores}.dmg"
```
Comment (reitermarkus) [inline Casks/beersmith2.rb]: I think you can leave this as `https`.

**Comment by reitermarkus** [comment]
The comment doesn't match the URL anymore, that's why it failed.  [When URL and homepage hostnames differ](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment), there has to be a comment in a specific format above the `url` stanza.

**Comment by jawshooah** [comment]
Thank you for the contribution. It needed some fixes, so they were made in bfa0c15c32d373b25cfc098959c1ebe8e09fd3cb. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by welly** [comment]
Thanks kindly for the info! I'll know for next time.

**Pull request closed without merging**